metadata/src/main/resources/common/metadata/IsrChange.json

```diff
@@ -0,0 +1,18 @@
+{
+  "apiKey": 4,
+  "type": "metadata",
+  "name": "IsrChangeRecord",
+  "validVersions": "0",
+  "fields": [
+    { "name": "PartitionId", "type": "int32", "versions": "0+", "default": "-1",
+      "about": "The partition id." },
+    { "name": "TopicId", "type": "uuid", "versions": "0+",
```

Comment:
q, how would this field get generated?

Reply:
The controller generates it (possibly randomly with a collision check, although the collision check isn't really necessary with a 128-bit space)